generator/src/main/java/line/bot/generator/PythonNextgenCustomClientGenerator.java

```diff
@@ -1368,7 +1368,7 @@ private ModelsMap postProcessModelsMap(ModelsMap objs) {
                     modelsToImport.add("from " + packageName + ".models." + underscore(modelImport) + " import " + modelImport);
                 }
 
-                model.getVendorExtensions().putIfAbsent("x-py-model-imports", modelsToImport);
+                model.getVendorExtensions().put("x-py-model-imports", modelsToImport);
```

Comment:
The method `putIfAbsent` does nothing because the value is set in [AbstractPythonCodegen.java at line 1022](https://github.com/OpenAPITools/openapi-generator/blob/v7.9.0/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonCodegen.java#L1022), which is called by [PythonNextgenCustomClientGenerator.java at line 1085](https://github.com/line/line-bot-sdk-python/blob/854107d10d33507a9008390a62936fe269e1597b/generator/src/main/java/line/bot/generator/PythonNextgenCustomClientGenerator.java#L1085).

Reply:
The other parts will be fixed in https://github.com/line/line-bot-sdk-python/pull/703.